### PR TITLE
[CoreBundle] Domain Name Validator

### PR DESCRIPTION
**Pull request by ngodfraind**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Fixed tickets | #1195

Validator added for the domain name.

### CODE REVIEW

File: main/core/Validator/Constraints/DomainName.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+namespace Claroline\CoreBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class DomainName extends Constraint
+{
+    public $message = 'invalid_domain_name';
```
Comment (stefk) [inline main/core/Validator/Constraints/DomainName.php]: A translation is probably needed for this message.

Comment (ngodfraind) [inline main/core/Validator/Constraints/DomainName.php]: Already done.

Comment (stefk) [inline main/core/Validator/Constraints/DomainName.php]: There's really something wrong with the new review system...

**Comment by Elorfin** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**